<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>3</y>
     <width>61</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>357</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 1</string>
    </attribute>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>357</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Author</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>501</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Credits</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>License</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Maintainer</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Apache License 2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Artistic License/GPL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Eclipse Plugin License 1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GNU General Public License v2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GNU General Public License v3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GNU Lesser General Public License</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MIT  License</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mozilla Public License 1.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New BSD License</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>This will automatically fill new files you create with the Details Given Below</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Terminal</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Android</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SDK:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NDK:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Logcat Filters:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Squirrel</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpreter:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>EmoFrameWork</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Python</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpreter:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>C</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpreter:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>C++</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpreter:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_9">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <action name="actionApply">
   <property name="text">
    <string>apply</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>actionApply</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
